.github/workflows/yocto-build-deploy.yml

```
@@ -503,6 +503,7 @@ jobs:
       # Separate "flasher" and "raw" variants are not used in the testing flow
       - name: Upload artifacts
         uses: actions/upload-artifact@26f96dfa697d77e81fd5907df203aa23a56210a8 # v4.3.0
+        if: github.event.pull_request.state != 'closed'
```

```suggestion
        if: github.event_name != 'push'
```
This workflow doesn't listen for PR close events, so your existing IF condition would be no-op and we would still upload on tag events, which we don't want IIRC.

---

I see what you mean, the called workflow doesn't listen to the `pull_request` closed event since that's for the caller workflow isn't it.